1_fetch/src/get_nwis_data.R

```diff
@@ -1,30 +1,25 @@
 
-download_nwis_data <- function(site_nums = c("01427207", "01432160", "01435000", "01436690", "01466500", "01477050"), parameterCd = '00010', dl_dir = '1_fetch/out'){
+download_nwis_data <- function(site_id, parameterCd = '00010', dl_dir = '1_fetch/out'){
   
   # create the file names that are needed for download_nwis_site_data
   # tempdir() creates a temporary directory that is wiped out when you start a new R session; 
   # replace tempdir() with "1_fetch/out" or another desired folder if you want to retain the download
-  download_files <- file.path(dl_dir, paste0('nwis_', site_nums, '_data.csv'))
+  download_file <- file.path(dl_dir, paste0('nwis_', site_id, '_data.csv'))
   
+  download_nwis_site_data(download_file, parameterCd = parameterCd)
 
-  # loop through files to download 
-  for (download_file in download_files){
-    download_nwis_site_data(download_file, parameterCd = parameterCd)
-    
-  }
-  return(download_files)
   
 }
 
-nwis_site_info <- function(fileout, nwis_data_path = '2_process/out/nwis_data.csv'){
+
+nwis_site_info <- function(fileout, input_data){
   
-  site_data <- read_csv(nwis_data_path)
+  site_data <- input_data
```

Comment:
perhaps simpler to change the argument to this function to `site_data` or replace usage of `site_data` in the function body below with `input_data`, but this works.